types/sparql-http-client/sparql-http-client-tests.ts

```diff
@@ -1,4 +1,4 @@
-import { SparqlHttp } from 'sparql-http-client';
+import SparqlHttp from 'sparql-http-client';
```

Comment:
Since this package has not default this import is incorrect as this syntax is used for `default`.